packages/ember-toucan-form/src/-private/radio-group-field.hbs

```diff
@@ -0,0 +1,18 @@
+<@form.Field @name={{@name}} as |field|>
+  <Form::Fields::RadioGroup
+    @label={{@label}}
+    @hint={{@hint}}
+    @error={{this.mapErrors field.rawErrors}}
```

Comment:
Same question as https://github.com/CrowdStrike/ember-toucan-core/pull/135/files#r1170298054

How come we can't use the `component` helper to pass these args along?

Reply:
(Answered! ✅ )